### DISCUSSION TITLE
react-native-sdk + auth0 (email passwordless)

### DISCUSSION THREAD

**Discussion by tmcyrix** [Web3Auth/Web3Auth#730] [Help]
Hey folks! I am currently implementing the react-native-sdk along with auth0 as provider with email-passwordless as auth flow and am currently stuck and hope to find help here.

**The current problem:**
Although I specify jwt in the loginConfig, the popup window still appears which should not happen according to the documentation.

**What I would like to achieve:**
I want to pass the idToken from auth0 to the web3auth sdk and without the popup window appearing.

**This is what my code currently looks like:**
```
// Initialize the Web3Auth.
const web3auth = new Web3Auth(WebBrowser, {
  clientId: "clientId-from-plugAndPlay",
  network: OPENLOGIN_NETWORK.TESTNET,
  loginConfig: {
    jwt: {
      name: "Email-Passwordless login with Auth0",
      verifier: "verifierId-from-customAuth",
      typeOfLogin: "jwt",
      clientId: "auth0-clientId",
    },
  },
});

// Start the login.
const state = await web3auth.login({
  loginProvider: LOGIN_PROVIDER.JWT,
  curve: "ed25519",
  extraLoginOptions: {
    domain: "https://auth0-domainUrl",
    login_hint: "email-from-user",
    verifierIdField: "sub",
    id_token: auth0Result.idToken,
    isVerifierIdCaseSensitive: false,
  },
});
```

**Comment by shahbaz17** [comment]
Hey @tmcyrix 

In your `extraLoginOptions`, you're mixing two logins. Email Passwordless and JWT.

```js
extraLoginOptions: {
    domain: "https://auth0-domainUrl",
    login_hint: "email-from-user", // use this, when you're to use Email Passwordless
    verifierIdField: "sub",
    id_token: auth0Result.idToken, // use this, when you're to use JWT login.
    isVerifierIdCaseSensitive: false,
  },
 ```
 
 Please check the SDK reference https://web3auth.io/docs/sdk/react-native/custom-authentication#extraloginoptions-for-special-login-methods to know more.
 
 Also, can you share from where you're getting `auth0Result`?
